man/man5/puppet.conf.5

```diff
@@ -453,6 +453,14 @@ Boolean; whether to generate the default schedule resources\. Setting this to fa
 .
 .IP "" 0
 .
+.SS "deviceconfdir"
```

Comment:
do we merge up manpages?

Reply:
it was auto-merged, it should not matter since it will be regenerated for each version